### PR TITLE
update README

### PR DESCRIPTION
**Pull request by rhagarty**
@ScrapCodes - please review

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+## Contributing In General
+
+Our project welcomes external contributions! If you have an itch, please
+feel free to scratch it.
```
Comment (ScrapCodes) [inline CONTRIBUTING.md]: ~If you have an itch, please feel free to scratch it.~. This sentence seems redundant.

**Comment by ScrapCodes** [comment]
@rhagarty Overall changes look, I have added comments, but they are not blockers. Please feel free to merge.

File: CONTRIBUTING.md (hunk 1 above)
Comment (rhagarty) [reply CONTRIBUTING.md]: This is boiler-plate stuff added to all journeys.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+For more details, see the [MAINTAINERS](MAINTAINERS.md) page.
+
+## Communication
+
+Please feel free to connect with us: [here](https://github.com/IBM/kafka-streaming/issues)
+
+## Setup
+
+Please add any special setup instructions for your project to help the
```
[the remark this answers is not in the document]
Comment (rhagarty) [reply CONTRIBUTING.md]: See my response above...

**Comment by rhagarty** [comment]
I will merge.

**Pull request merged**